_data/sites.json

```diff
@@ -6227,6 +6227,17 @@
         ]
     },
 
+    {
+        "name": "Miro",
+        "url": "https://miro.com/app/settings/user-profile/",
+        "email": "support@mister-wong.de",
```

Comment:
```suggestion
```

If the email is not needed for deletion, then we can omit it. (And the domain looks unrelated to the website ?)

Reply:
@sebix can you please help here?